### PR TITLE
Remove extra parenthesis

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

**Comment by tdykstra** [comment]
@Youssef1313 I don't see in the diff where an extra parenthesis is removed.

File: docs/core/tutorials/testing-library-with-visual-studio.md [hunk 1]
```diff
@@ -67,11 +67,11 @@ To create the unit test project, do the following:
 
    The source code created by the unit test template does the following:
 
-   * It imports the <xref:Microsoft.VisualStudio.TestTools.UnitTesting?displayProperty=nameWithType> namespace, which contains the types used for unit testing.
+   - It imports the <xref:Microsoft.VisualStudio.TestTools.UnitTesting?displayProperty=nameWithType> namespace, which contains the types used for unit testing.
 
-   * It applies the <xref:Microsoft.VisualStudio.TestTools.UnitTesting.TestClassAttribute>) attribute to the `UnitTest1` class. Each test method in a test class tagged with the <xref:Microsoft.VisualStudio.TestTools.UnitTesting.TestMethodAttribute> attribute is executed automatically when the unit test is run.
+   - It applies the <xref:Microsoft.VisualStudio.TestTools.UnitTesting.TestClassAttribute> attribute to the `UnitTest1` class. Each test method in a test class tagged with the <xref:Microsoft.VisualStudio.TestTools.UnitTesting.TestMethodAttribute> attribute is executed automatically when the unit test is run.
```
Comment (Youssef1313) [inline docs/core/tutorials/testing-library-with-visual-studio.md]: @tdykstra , It's here, between `>` and `attribute`

File: docs/core/tutorials/testing-library-with-visual-studio.md [hunk 2]
```diff
@@ -87,7 +87,7 @@ To create the unit test project, do the following:
 
 When Visual Studio runs a unit test, it executes each method marked with the <xref:Microsoft.VisualStudio.TestTools.UnitTesting.TestMethodAttribute> attribute in a unit test class, the class to which the  <xref:Microsoft.VisualStudio.TestTools.UnitTesting.TestClassAttribute> attribute is applied. A test method ends when the first failure is encountered or when all tests contained in the method have succeeded.
 
-The most common tests call members of the <xref:Microsoft.VisualStudio.TestTools.UnitTesting.Assert> class. Many assert methods include at least two parameters, one of which is the expected test result and the other of which is the actual test result. Some of its most frequently called methods are shown in the table below.
+The most common tests call members of the <xref:Microsoft.VisualStudio.TestTools.UnitTesting.Assert> class. Many assert methods include at least two parameters, one of which is the expected test result and the other of which is the actual test result. Some of its most frequently called methods are shown in the table below:
```
Comment (mairaw) [inline docs/core/tutorials/testing-library-with-visual-studio.md]: small suggestion to not use below: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/b/below

```suggestion
The most common tests call members of the <xref:Microsoft.VisualStudio.TestTools.UnitTesting.Assert> class. Many assert methods include at least two parameters, one of which is the expected test result and the other of which is the actual test result. Some of its most frequently called methods are shown in the following table:
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @Youssef1313. Can you also make one additional change? I left the suggestion.

**Review by mairaw — APPROVED** [review]

**Pull request merged**